datasets/covid19_google_mobility_eu/dataset.yaml

```diff
@@ -0,0 +1,46 @@
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+dataset:
+  name: covid19_google_mobility_eu
+  friendly_name: covid19_google_mobility_eu
+  description: |-
+    Terms of use
+    In order to download or use the data or reports, you must agree to the Google Terms of Service: https://policies.google.com/terms
+    Description
+    This dataset aims to provide insights into what has changed in response to policies aimed at combating COVID-19. It reports movement trends over time by geography, across different categories of places such as retail and recreation, groceries and pharmacies, parks, transit stations, workplaces, and residential.
+    This dataset is intended to help remediate the impact of COVID-19. It shouldn’t be used for medical diagnostic, prognostic, or treatment purposes. It also isn’t intended to be used for guidance on personal travel plans.
+    To learn more about the dataset, the place categories, and how we calculate these trends and preserve privacy, do the following:
+    • Visit the help center: https://support.google.com/covid19-mobility.
+    • Or, read the dataset documentation: https://www.google.com/covid19/mobility/data_documentation.html.
+  dataset_sources: ~
+  terms_of_use: ~
+
+
+resources:
+  - type: bigquery_dataset
+    dataset_id: covid19_google_mobility_eu
+    description: |-
+        Terms of use
+        In order to download or use the data or reports, you must agree to the Google Terms of Service: https://policies.google.com/terms
+        Description
+        This dataset aims to provide insights into what has changed in response to policies aimed at combating COVID-19. It reports movement trends over time by geography, across different categories of places such as retail and recreation, groceries and pharmacies, parks, transit stations, workplaces, and residential.
+        This dataset is intended to help remediate the impact of COVID-19. It shouldn’t be used for medical diagnostic, prognostic, or treatment purposes. It also isn’t intended to be used for guidance on personal travel plans.
+        To learn more about the dataset, the place categories, and how we calculate these trends and preserve privacy, do the following:
+        • Visit the help center: https://support.google.com/covid19-mobility.
+        • Or, read the dataset documentation: https://www.google.com/covid19/mobility/data_documentation.html.
+  - type: storage_bucket
+    name: covid19-mobility-eu
+    uniform_bucket_level_access: True
+    location: US
```

Comment:
You can remove these lines because your pipelines don't use a bucket

Reply:
As discussed, let's put this bucket back so that the load operator will work (source bucket and target BQ table need to be in the same region: US)